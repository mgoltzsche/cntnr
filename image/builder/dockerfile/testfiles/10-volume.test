FROM alpine:3.7
VOLUME /var/app
VOLUME /var/app /data/dir

# ASSERT CFG /config/Volumes/~1data~1dir=map[]
# ASSERT CFG /config/Volumes/~1var~1app=map[]