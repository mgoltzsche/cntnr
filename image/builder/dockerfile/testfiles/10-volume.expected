FROM alpine:3.7
VOLUME "/var/app"
VOLUME "/var/app" "/data/dir"